concrete/src/Updater/Migrations/Migrations/Version20180710203437.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Concrete\Core\Updater\Migrations\Migrations;
+
+use Concrete\Core\Updater\Migrations\AbstractMigration;
+use Concrete\Core\Updater\Migrations\RepeatableMigrationInterface;
+use Doctrine\DBAL\Schema\Schema;
+use Concrete\Core\Entity\Express\Entity;
+
+class Version20180710203437 extends AbstractMigration implements RepeatableMigrationInterface
+{
+
+    public function upgradeDatabase()
```

Comment:
Should we add a description field to these things that makes it easy to see why a migration was added?